skimage/draw/_random_shapes.py

```diff
@@ -117,8 +117,8 @@ def _generate_triangle_mask(point, image, shape, random):
 
     Parameters
     ----------
-    point : tuple
-        The row and column of the top left corner of the rectangle.
+    point : tuple                                                  .__
+        The row and column of the top left corner of the triangle:  \/
```

Comment:
This looks dirty :)

Reply:
Agree. It looked good in Vim though - where the gap between the lines wasn't as wide as here. :)